test/lit.cfg.py

```diff
@@ -57,14 +57,11 @@
 
 if config.spirv_tools_have_spirv_as:
     llvm_config.add_tool_substitutions(['spirv-as'], [config.spirv_tools_bin_dir])
-    config.available_features.add('spirv-as')
     using_spirv_tools = True
 
 if config.spirv_tools_have_spirv_val:
     llvm_config.add_tool_substitutions(['spirv-val'], [config.spirv_tools_bin_dir])
```

Comment:
Should we add `spirv-as` and `spirv-val'` with a single call to `add_tool_substitutions()`, as per `llvm-as`, `llvm-dis`, etc.?

Should we also rename `tools` to `llvm_tools`?